src/app/App.js

```diff
@@ -0,0 +1,25 @@
+import './App.css';
+import { Counter } from "../shared/ui/Counter/Counter";
+import { GenreSelect } from "../shared/ui/GenreSelect/GenreSelect";
+import { SearchForm } from "../shared/ui/SearchForm/SearchForm";
```

Comment:
<img width="175" alt="image" src="https://github.com/GaKhalatyan86/ReactLessons/assets/36797396/5664a68c-2a32-457f-8b0a-cc0973804418">

to avoid name duplication in imports we can create an `index.js` inside shared/ui folder and export your components like:
`export { SelectForm } from './SelectForm';`

here is an example:
https://github.com/yesnoruly/loripsum-generator/tree/master/src/shared/ui

Reply:
changed